### ISSUE TITLE
Continuous Integration

### ISSUE THREAD

**Issue by YannickDieter** [SiLab-Bonn/pymosa#12]
I just realized that we dont have a CI for pymosa, as for all other projects.
I think it would be good if we add this?

**Comment by YannickDieter** [comment]
We need unit tests for this right? ;)

**Comment by DavidLP** [comment]
Yes ;-) Not sure if @themperek thinks if it makes sense to setup cosimulation

**Comment by themperek** [comment]
I think it is good idea and not so difficult.

**Comment by laborleben** [comment]
I want to add: CI of firmware and readout.
